frontends/web/src/utils/request.js

```diff
@@ -53,6 +66,11 @@ function handleError(endpoint) {
 }
 
 export function apiGet(endpoint) {
+    // if apiGet() is invoked imediatelly this can error with:
+    // request.js:64 Uncaught TypeError: Cannot read properties of undefined
+    // (reading 'runningInQtWebEngine')
```

Comment:
@x1ddos helped me figure out how cyclic i18n<->request dependencies caused this errored

Reply:
imediatelly -> immediately

Reply:
With `i18n` I meant `frontends/web/src/i18n/i18n.js`, here:

- `src/i18n/i18n.js` depends on `src/utils/request.js` (reason: get native-locale from the backend)
- `src/utils/request.js` depends on `src/i18n/i18n` (reason: to translate generic error alert, IMO the error should not be handled in the request module)